src/CMakeLists.txt

```diff
@@ -530,7 +530,7 @@ set_target_properties(ceph-osd PROPERTIES
   INSTALL_RPATH "")
 install(TARGETS ceph-osd DESTINATION bin)
 
-if (WITH_CEPHFS)
+if (WITH_CEPHFS OR (FREEBSD AND WITH_FUSE))
```

Comment:
i don't see a good reason adding FREEBSD here.

Reply:
yeah, it sounds like these changes will allow us to remove the `-D WITH_CEPHFS=OFF` from do_freebsd.sh and avoid more special-cases around `FREEBSD`

Reply:
I'd like to have as little as possible exceptions, because they tend to confuse others when adding more code. And getting ride of  `-D WITH_CEPHFS=OFF` would be a good bonus.